src/vault/client/http.clj

```diff
@@ -164,6 +164,22 @@
            :as :json})))))
 
 
+(defn- authenticate-ldap!
+  "Updates the token ref by making a request to authenticate with a username
+  and password, to be authenticated against an LDAP backend."
+  [client credentials]
+  (let [{:keys [username password]} credentials]
+    (api-auth!
+      (str "user " username)
```

Comment:
The string here should be `"ldap "` to distinguish this from the username/password auth method.

Reply:
Done!